nuto/mechanics/elements/DofElementFem.h

```diff
@@ -39,22 +37,22 @@ class ElementFem : public ElementInterface
         return nodeValues;
     }
 
-    Eigen::MatrixXd GetNMatrix(NaturalCoords ipCoords) const override
+    virtual Eigen::MatrixXd GetNMatrix(NaturalCoords ipCoords) const override
```

Comment:
All the added `virtual`s here are unnecessary. We know it's a virtual function because of `override`.